_episodes/32.md

```diff
@@ -0,0 +1,383 @@
+---
+layout: episode
+title:  "32: Trino Tardigrade: Try, try, and never die"
+date: 2021-11-20
+tags: architecture mpp fault tolerance
+youtube_id: ""
+wistia_id: ""
+sections: 
+   - title: "Concept of the month"
+     desc: "Introducing Project Tardigrade"
+     time: 0
+   - title: "Demo of the month"
+     desc: "Task retries with Project Tardigrade"
+     time: 0
+   - title: "PR of the month"
+     desc: "PR of the month: PR Trino lineage fails for `AliasedRelation`"
+     time: 0
+   - title: "Question of the month"
+     desc: "How do you cast JSON to varchar with Trino?"
+     time: 0
+---
+
+## Guests
+
+ * Andrii Rosa, Software Engineer at [Starburst](https://www.starburst.io)
+  ([@andrii-rosa-79578561](https://www.linkedin.com/in/andrii-rosa-79578561/)).
+ * Brian Zhan, Product Manager at [Starburst](https://www.starburst.io)
+  ([@brianzhan1](https://twitter.com/brianzhan1)).
+ * Lukasz Osipiuk, Software Engineer at [Starburst](https://www.starburst.io)
+  ([@losipiuk](https://twitter.com/losipiuk)).
+ * Martin Traverso, Trino & Presto Co-founder and CTO at [Starburst](https://www.starburst.io)
+ ([@mtraverso](https://twitter.com/mtraverso)).
+ * Martin Traverso, Software Engineer at [Starburst](https://www.starburst.io)
+ ([@linzebing](https://www.linkedin.com/in/linzebing/)).
+ 
+## Trino Summit 2021
+
+If you missed [Trino Summit 2021](https://www.starburst.io/resources/trino-summit/),
+you can watch it on demand, for free!
+ 
+## Releases 367 and 368
+​
+Martin's official announcements merged into one:
+​
+* Lineage tracking for `WITH` clauses and subqueries.
+* Option to hide inaccessible columns in `SELECT *`.
+* `flush_metadata_cache()` procedure for the Hive connector.
+* Improve performance of `DECIMAL` type.
+* File-based access control for the Iceberg connector.
+* Support for `TIME` type in the SingleStore connector.
+* Support for `BINARY` type in the Phoenix connector.
+​
+Manfred's additional notes:
```

Comment:
Check why this dies bit seen to render with empty line above and below.

Reply:
Fixed, it was a strange character